frontend/storybook/src/components/entry/publish/publish-one.ts

```diff
@@ -89,9 +88,10 @@ export const PublishFullOne = (props?:PublishArgs) => {
         <div slot="pills">${PillClose({contents:STR_PILL})}</div>
         <div slot="pills">${PillClose({contents:STR_PILL})}</div>
         <title-ji slot="column_four" color="blue" size="medium" weight="normal">${STR_RESOURCES}</title-ji>
-        <title-wicon title="${STR_ICONLABEL}" path="${STR_CHECKBOX}" ${uploaded && "uploaded"} slot="column_four"></title-wicon>
-        <title-wicon title="${STR_ICONLABELTWO}" path="${STR_ADD}"  slot="column_four"></title-wicon>
-
+        <title-wicon title="${STR_LESSONPLAN}" path="${STR_ADD}"  ${uploaded && "uploaded"} slot="column_four"></title-wicon>
```

Comment:
these can all be grouped into a single element and set with a single "uploaded" property - that will also move the need for the static text into the element instead of the component

Reply:
Fixed